src/views/LoginView/LoginView.scss

```diff
@@ -1,8 +1,13 @@
-.loginColumn {
-  position: absolute;
-  top: 20%;
-  right: 0;
-  left: 0;
-  max-width: 340px;
-  margin: auto;
-}
\ No newline at end of file
+html,
+body {
+  background: lightgray;
+}
+
+.LoginView {
+  &__grid {
+    height: 80vh;
```

Comment:
view height, there is also view width.  percentages correspond to the size of the parent whereas this corresponds to the actual view.